spec.md

```diff
@@ -1657,7 +1658,12 @@ This RPC allows the CO to expand the size of a volume.
 This call MAY be made by the CO during any time in the lifecycle of the volume after creation if plugin has `VolumeExpansion.ONLINE` capability.
 If plugin has `EXPAND_VOLUME` node capability, then `NodeExpandVolume` MUST be called after successful `ControllerExpandVolume` and `fs_resize_required` in `ControllerExpandVolumeResponse` is `true`.
 
-If the plugin has only `VolumeExpansion.OFFLINE` expansion capability and volume is currently controller-published to a node , `ControllerExpandVolume` MUST be called ONLY after succesful `ControllerUnpublishVolume`.
+If the plugin has only `VolumeExpansion.OFFLINE` expansion capability and volume is currently published or available on a node - `ControllerExpandVolume` MUST be called ONLY after either:
+- Successful completion of `ControllerUnpublishVolume` if plugin has `PUBLISH_UNPUBLISH_VOLUME` controller capability.
+OR
+- Successful completion of `NodeUnstageVolume` if volume was made available on the node via `NodeStageVolume` and plugin does not have `PUBLISH_UNPUBLISH_VOLUME` controller capability.
```

Comment:
> if volume was made available on the node via `NodeStageVolume`

The CO cannot reason about this, it cannot know the implementation details of the plugin